ci/aws-lambda-ruby-dev/3.3/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:latest
```

Comment:
You only have one FROM line to specify the base image, this one has snuck in since the previous version.

Reply:
Ooops, fixed!